frontend/__tests__/src/components/Metrics/ReportStep/ExpiredDialog.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, getByText, queryByText, render, waitFor } from '@testing-library/react'
+import { setupStore } from '../../../utils/setupStoreUtil'
+import { ExpiredDialog } from '@src/components/Metrics/ReportStep/ExpiredDialog'
+import { Provider } from 'react-redux'
+import userEvent from '@testing-library/user-event'
+
+describe('ExpiredDialog', () => {
+  it('should show expired dialog when Given isExpired is true And not show expired dialog when Click No button', async () => {
```

Comment:
Update the description from the aspect of the user, not dev

Reply:
done